tools/shared.py

```diff
@@ -1899,8 +1900,6 @@ def link_lld(args, target, opts=[], lto_level=0):
         '--import-memory',
         '--export',
         '__wasm_call_ctors',
-        '--export',
```

Comment:
Why remove this?

Reply:
It was redundant.. but I reverted that part to keep this change clear.